agtern/server/database/crud.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy.orm import Session
+
+from . import models
+from ...common import schemas
+
+
+def get_internship(db: Session, internship_id: int) -> schemas.Internship:
+    """Gets an internship by internship_id."""
+    return db.query(models.Internship)\
+        .filter(models.Internship.id == internship_id)\
+        .first()
+
+def get_all_internships(db: Session, skip: int = 0, limit: int = 100) -> list[schemas.Internship]:
+    """Gets all internships. Use skip and limit to offset and limit number of results."""
+    return db.query(models.Internship)\
+        .offset(skip)\
+        .limit(limit)\
+        .all()
+
+def create_internship(db: Session, internship: schemas.InternshipCreate):
+    """Creates an internship in the database."""
+
+    db_internship = models.Internship(**internship.dict())
```

Comment:
```suggestion
    try:
        db_internship = models.Internship(**internship.dict())
    except ValidationError as errors:
        LOG.error("Could not create internship!")
        LOG.error(errors)
        return None
```
If `Internship` validation fails, it doesn't say why. You need to catch the validation errors and log them. You could either deal with the `None` return value when you use this function, or you could catch the validation errors where you use this function instead.